frictionless/fields/integer.py

```diff
@@ -36,6 +36,10 @@ def value_reader(cell):
             if isinstance(cell, str):
                 if pattern:
                     cell = pattern.sub("", cell)
+                # for numeric string starting with zero for example, 001, 00, 01
+                has_leading_zero = re.match(r"0\d+", cell.strip())
```

Comment:
Let's not use regural expressions here and in the number as it might hurt performance. 
This line will be executed 1 000 000 times for a numeric table with e.g. 100 x 10 000 dimension